AdyenActions/Components/QRCode/QRCodeComponent.swift

```diff
@@ -30,6 +36,11 @@ internal final class QRCodeComponent: ActionComponent, Localizable {
     /// - Parameter style: The component UI style.
     public init(style: QRCodeComponentStyle = QRCodeComponentStyle()) {
         self.style = style
+        
+        self.timeLeft = timeoutInterval
```

Comment:
this also can be:

```
var timeLeft: TimeInterval = timeoutInterval
```

Reply:
i don't think this is possible. properties are initalized before `self` is available, we can't use a property to initialize another one, unless it is a lazy one.